pkgs/by-name/gi/git-identity/package.nix

```
@@ -0,0 +1,45 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  ronn,
+  nix-update-script,
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "git-identity";
+  version = "1.1.1";
+
+  src = fetchFromGitHub {
+    owner = "madx";
+    repo = "git-identity";
+    rev = "v${version}";
```

```suggestion
    rev = "refs/tags/v${version}";
```
Prevents eventual branch/tag mismatch.